### ISSUE TITLE
CHADdeps fails on ensurepip

### ISSUE THREAD

**Issue by tsjordan-eng** [ms-jpq/chadtree#190]
I followed the Plug installation instructions. python3.8 virtualenv is installed. Installlation fails.

When I run :CHADdeps I get:
```
Error detected while processing function CHADon_stderr:
line    1:
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/tsjorda/dotfiles/neovim/.config/nvim/plugged/chadtree/chadtree/__main__.py", line 65, in <module>
    EnvBuilder(
  File "/usr/lib/python3.8/venv/__init__.py", line 68, in create
    self._setup_pip(context)
  File "/usr/lib/python3.8/venv/__init__.py", line 289, in _setup_pip
    subprocess.check_output(cmd, stderr=subprocess.STDOUT)
  File "/usr/lib/python3.8/subprocess.py", line 411, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/usr/lib/python3.8/subprocess.py", line 512, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/home/tsjorda/dotfiles/neovim/.config/nvim/plugged/chadtree/.vars/runtime/bin/python3.8', '-Im', 'ensurepip', '--upgrade', '--default-pip']'
returned non-zero exit status 1.
```

When I re-run then call `:message`:
```
/home/tsjorda/dotfiles/neovim/.config/nvim/plugged/chadtree/.vars/runtime/bin/python3: No module named pip
Error detected while processing function CHADon_stderr:
line    1:
Installation failed, check :message
Error detected while processing function CHADon_exit:
line    1:
 | CHADTree EXITED - 1
```

I'm not sure why `EnvBuilder` is not installing pip into the venv, but it looks like that's the blocking issue.

Any tips?

Note: I do use a proxy and have to trust the python.org url

The following is in my pyenv.cfg:
```cfg
home = /usr/bin
include-system-site-packages = false
version = 3.8.7
```

**Comment by JonasRSV** [comment]
I also have this error, CHADtree broke after upgrading

A temporary solution that worked from me was to just remove the .vars/runtime directory. Then create a virtualenv called 'runtime' that contains pip.

**Comment by ngocthienvnn** [comment]
I had the same issue today and solved it by those steps(on NVIM v0.5.0 -  Kubuntu 20.04.2): 
1. Remove chadtree/.vars
2. sudo apt install python3.8-venv
3. open Nvim and run :CHADdeps.

**Comment by ms-jpq** [comment]
should be fixed now i think ill close it for now
